src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -21,23 +21,54 @@ object Lorre extends App with LazyLogging {
   lazy val db = DatabaseUtil.db
   val tezosNodeOperator = new TezosNodeOperator(TezosNodeInterface)
 
-  try {
-    while(true) {
-      logger.info("Fetching blocks")
-      processTezosBlocks()
-      logger.info("Fetching accounts")
-      processTezosAccounts()
-      logger.info("Taking a nap")
-      Thread.sleep(sleepIntervalInSeconds * 1000)
-    }
-  } finally db.close()
+  val usage =
+    """
+      Usage: runMain tech.cryptonomic.conseil.Lorre --platform <platform> --network <network>
+    """.stripMargin
+
+  val argsList = args.toList
+  val (platform, network) = argsList match {
+    case "--platform" :: pl :: "--network"  :: nw :: Nil => (pl, nw)
+    case "--network"  :: nw :: "--platform" :: pl :: Nil => (pl, nw)
+    case _ =>
+      println(usage)
+      sys.exit(1)
+  }
+
+  val supportedPlatformsAndNetworks =
+    """
+      | The supported platforms are: tezos.
+      | The supported networks are: zeronet.
```

Comment:
This should not be hard coded. Please read it from the conf.

Reply:
Okay, will do.